features/api_token_info.feature

```diff
@@ -0,0 +1,113 @@
+Feature: api.u.pro.token_info.v1
+
+  @uses.config.contract_token
```

Comment:
Let's add an integration test where we try to see an expired token. We can use the staging expired token for this.

Reply:
Apologies for the oversight, only added a unit test for this. An integration test too would make sense. Will add it

Reply:
@lucasmoura Updated with the new integration test and added error messages/exceptions specific to token_info and simulate status too